src/osd/scrubber/pg_scrubber.h

```diff
@@ -3,6 +3,70 @@
 
 #pragma once
 
+// clang-format off
+/*
+
+Main Scrubber interfaces:
+
+┌──────────────────────────────────────────────┬────┐
+│                                              │    │
+│                                              │    │
+│       PG                                     │    │
+│                                              │    │
+│                                              │    │
+├──────────────────────────────────────────────┘    │
+│                                                   │
+│       PrimaryLogPG                                │
+└────────────────────────────────┬──────────────────┘
+                                 │
+                                 │
+                                 │ ownes & uses
+                                 │
+                                 │
+                                 │
+┌────────────────────────────────▼──────────────────┐
+│             <<ScrubPgIf>>                         │
```

Comment:
ScrubPgIF

Reply:
<< >> seems to be the recommended UML notation?

Reply:
Sorry, I mean that the 'f' in ScrubPgIF is lower case rather than upper case as it is in the code.